bin/graftM

```diff
@@ -145,7 +151,7 @@ if __name__ == '__main__':
     
     logging_options = graft_parser.add_argument_group('logging options')
     logging_options.add_argument('--verbosity', metavar='verbosity', help='1 - 5, 1 being silent, 5 being noisy indeed. Default = 4', type=int, default=4)
-    logging_options.add_argument('--log', metavar='logfile', help='Output logging information to file', type=str, default=False)
+    logging_options.add_argument('--log', metavar='logfile', help='Output logging information to file', default=False)
```

Comment:
don't wory about it now, but there's the concept of parent in argparse I think which might help reduce the duplication of these arguments